client/post.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <script defer src="front.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Journ.al</title>
+</head>
+
+<body>
+    <section class = 'post flex-container'>
```

Comment:
you added this on a new page, however how does the user get from one page to the other? 

you did add your submit button to have a function to guide it to this page but stated it was incorrect, I don’t know if it would be correct to instead write <a href="client/post.html" ><button type="submit" id="publish">📰 Publish</button></a> - I don’t know if you tried this. it would work on live server defo but I am unsure if it would when you eventually host it

also going back to the navigation, that was the only navigation I could see. potentially adding a logo that could navigate you to the original page or a nav bar with "create post" or "view posts" could be good ?